bower.json

```diff
@@ -17,13 +17,15 @@
     "tests"
   ],
   "dependencies": {
-    "angular": "^1.5.3",
-    "angular-animate": "^1.5.3",
+    "angular": "1.5.5",
```

Comment:
Btw switching back to `^1.5.3` solved as well previous comment about resolution :)